### ISSUE TITLE
Support interaction with tree service from `neofs-cli`

### ISSUE THREAD

**Issue by fyrchik** [nspcc-dev/neofs-node#1332]
- [x] `Add`
- [x] `AddByPath`
- [ ] `Remove`
- [ ] `Move`
- [x] `GetNodeByPath`
- [ ] `GetSubTree`
- [ ] `Healthcheck`
- [ ] `GetOpLog`
- [x] `ListTrees`

**Comment by carpawell** [comment]
All the Tree Service RPC should have corresponding CLI commands. `Apply` is an exception.

**Comment by acid-ant** [comment]
How about common command for the whole list of services on node? Like this:
```
@:~/workspace/neofs-node$ neofs-cli --rpc-endpoint 192.168.130.71:8080  node services
tree service status: N/A
@:~/workspace/neofs-node$ neofs-cli --rpc-endpoint 192.168.130.71:8080  node services
tree service status: available
@:~/workspace/neofs-node$

```

**Comment by vkarak1** [comment]
There are several suggestions / comments which could help to make life better to work with tree service:

1. It would be great to see details in neofs-storage service log regarding tree syncing, for now I was able to see only the following types of messages:
```
debug        tree/sync.go:280        syncing container trees...        {"cid": "5S1mE1yrhm7Ww3JEiWSazTwJbDCvDwyqYB6bSpmnmy9N
debug        tree/sync.go:291        container trees have been synced        {"cid": "5S1mE1yrhm7Ww3JEiWSazTwJbDCvDwyqYB6bSpmnmy9N"}
```
Adding details for example which tree / path and etc  has been synchronized will be great for testing purposes.
 2. I has been working on testing tree service where I turned on node which contained S3 container and I expected to see something useful in neofs-node logs (something like 'container_id / tree name / is updating / from which node' and etc), but there is no any mention about synchonization  of tree service of this S3 container.
 3. Since tree service is available only for S3, it would be great to allow `neofs-cli tree ` working with bucket name. It would be great to specify --cid OR --bm (bucket name) - where one of them is required.
